### PR TITLE
New message model

### PR DESCRIPTION
**Pull request by gvillalta99**
Updates the Message model to have only one field: body:text.

### CODE REVIEW

File: spec/validators/downcase_validator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    it 'sets an error message' do
+      expect(model).to receive(:errors)
+      expect(errors).to receive(:add).with('body', 'has uppercased letters.')
+      validator.validate_each(model, 'body', body)
+    end
+  end
+
+  describe 'valid attribute' do
+    let(:body) { 'body with uppercase letters' }
```
Comment (jmcoimbra) [inline spec/validators/downcase_validator_spec.rb]: He have repeated code here, bro. Just hold on, we can write it in a better way.
Also I think we can write it in a user friendly way, let me try to re-write it here...

Comment (gvillalta99) [inline spec/validators/downcase_validator_spec.rb]: Go ahead @jmcoimbra I'll wait for your changes, let me know if I can help you with anything...

**Comment by jmcoimbra** [comment]
I also would like to understand the "validator" idea (something new to me). Shouldn't we just have that in our model?

File: spec/validators/downcase_validator_spec.rb (hunk 1 above)
Comment (jmcoimbra) [inline spec/validators/downcase_validator_spec.rb]: Hey, sorry, just checked out your code and it has no repeated code. I would suggest a "before do" block in this case.

**Comment by gvillalta99** [comment]
@jmcoimbra I'm commeting here so I could answer your questions without losing the context.
The validator is an abstraction of the validation process, it encapsulates all the validation logic so we can use it as a component in any model.

It can be used in your model like this:

```ruby
class Model < ActiveRecord::Base
  validates :attribute,  downcase: true
end
```

**Pull request merged**